### PR TITLE
feat(holesky): deploy test cmtat

### PR DESCRIPTION
**Pull request by zarifpour**
```
+==================================+
|  Verifying Proxy Admin Contract  |
+==================================+

Proxy Admin contract address: 0x033cc0Fd48ECE2364EFfce9745Af055D0B8A339A 

Verifying contract on explorer...
The contract 0x033cc0Fd48ECE2364EFfce9745Af055D0B8A339A has already been verified on Etherscan.
https://holesky.etherscan.io/address/0x033cc0Fd48ECE2364EFfce9745Af055D0B8A339A#code
Verification submitted for contract at address: 0x033cc0Fd48ECE2364EFfce9745Af055D0B8A339A
Proxy Admin contract verified successfully.

+=====================================+
|  Verifying Implementation Contract  |
+=====================================+

Implementation contract address: 0x73225F88fEEA4E41Fc67E986a95AC61dd7118866 

Verifying contract on explorer...
Successfully submitted source code for contract
contracts/modules/CMTAT_BASE.sol:CMTAT_BASE at 0x73225F88fEEA4E41Fc67E986a95AC61dd7118866
for verification on the block explorer. Waiting for verification result...

Successfully verified contract CMTAT_BASE on the block explorer.
https://holesky.etherscan.io/address/0x73225F88fEEA4E41Fc67E986a95AC61dd7118866#code

Verification submitted for contract at address: 0x73225F88fEEA4E41Fc67E986a95AC61dd7118866
Implementation contract verified successfully.

+======================+
|  Deployment Summary  |
+======================+

Deployed Proxy Contract Address: 0xd8d42193150763b51a8C5bD1F372Cb0e6A75DCf9
Proxy Admin Contract Address: 0x033cc0Fd48ECE2364EFfce9745Af055D0B8A339A
Implementation Contract Address: 0x73225F88fEEA4E41Fc67E986a95AC61dd7118866
```

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -3,6 +3,7 @@ PRIVATE_KEY="0f3456f7f1ed59aaa29f35f4674a87e754e1055249a2888bdaec3dea49d2e456"
 MAINNET_NODE=""
 GOERLI_NODE=""
```
Comment (diego-G) [inline .env.example]: Let's remove GOERLI references.

File: .openzeppelin/unknown-17000.json [hunk 2]
```diff
@@ -0,0 +1,307 @@
+{
```
Comment (diego-G) [inline .openzeppelin/unknown-17000.json:1]: Shall we remove now *1337.json file?

File: hardhat.config.js [hunk 3]
```diff
@@ -57,6 +62,7 @@ module.exports = {
   etherscan: {
     apiKey: {
       mainnet: process.env.ETHERSCAN_API_KEY,
+      holesky: process.env.ETHERSCAN_API_KEY,
       goerli: process.env.ETHERSCAN_API_KEY,
```
Comment (diego-G) [inline hardhat.config.js]: Remove goerli everywhere please.

**Review by diego-G — CHANGES_REQUESTED** [review]

File: .openzeppelin/unknown-17000.json (hunk 2 above)
Comment (mariogutval) [reply .openzeppelin/unknown-17000.json:1]: I think we should add it to .gitignore. It's going to appear every time we simulate a deployment locally.

File: .env.example (hunk 1 above)
Comment (mariogutval) [reply .env.example]: Agree

File: hardhat.config.js [hunk 4]
```diff
@@ -33,6 +33,11 @@ module.exports = {
...
       gasPrice: 450000000000
     },
+    holesky: {
+      url: process.env.HOLESKY_NODE,
+      accounts: [process.env.PRIVATE_KEY],
+      chainId: 17000
+    },
     // mumbai: {
     //   url: process.env.MUMBAI_NODE,
     //   accounts: [process.env.PRIVATE_KEY],
```
Comment (mariogutval) [inline hardhat.config.js]: Should we also remove `mumbai`?

**Review by mariogutval — CHANGES_REQUESTED** [review]

File: .openzeppelin/unknown-17000.json (hunk 2 above)
Comment (zarifpour) [reply .openzeppelin/unknown-17000.json:1]: done, removed and added to .gitignore

File: hardhat.config.js (hunk 4 above)
Comment (zarifpour) [reply hardhat.config.js]: removed

**Review by diego-G — APPROVED** [review]

**Review by mariogutval — APPROVED** [review]

**Pull request merged**